### DISCUSSION TITLE
reviewdog: fail to get diff: failed to get merge-base commit: exit status 128

### DISCUSSION THREAD

**Discussion by ashunimbz** [reviewdog/reviewdog#1002] [Q&A]
I am facing this issue on gitlab 

reviewdog -f=staticcheck -name=staticcheck-discussion -reporter=gitlab-mr-discussion

This issue happens when feature branch is a older branch seems that it is somehow related to git shallow copy

**Comment by benford-servicem8** [comment]
I had the same issue and fixed by turning off git shallow copy as you suggested
